packages/esm-patient-chart-app/src/visit/visit-form/visit-form.scss

```
@@ -81,3 +81,23 @@
   margin-top: spacing.$spacing-04;
 }
 
+:global(.omrs-breakpoint-lt-desktop) {
+  .container {
+
+  & section {
+    display: flex;
+    gap: 4rem;
+
+    .sectionTitle {
+      flex-basis: 30%;
+      min-width: 8rem;
+      text-align: left;
+    }
+
+    .sectionField {
+      flex-basis: 70%;
+    }
+  }
+ }
+}
```

```suggestion
```suggestion
:global(.omrs-breakpoint-lt-desktop) {
  .container {
    & section {
      display: flex;
      gap: 4rem;

      .sectionTitle {
        flex-basis: 30%;
        min-width: 8rem;
        text-align: left;
      }

      .sectionField {
        flex-basis: 70%;
      }
    }
  }
}
```
```